### PR TITLE
Bump version of krew to v0.3.0

### PR DESCRIPTION
**Pull request by corneliusweig**
Release notes:
https://github.com/kubernetes-sigs/krew/releases/tag/v0.3.0

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **doesn't** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file plugins/krew.yaml is not a straightforward version bump: version should move forward ("v0.2.1" vs "v0.2.1-103-g0837beb")

File: plugins/krew.yaml [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 apiVersion: krew.googlecontainertools.github.com/v1alpha2
 kind: Plugin
 metadata:
   name: krew
 spec:
-  version: "v0.2.1"
-  homepage: https://sigs.k8s.io/krew
```
Comment (corneliusweig) [inline plugins/krew.yaml]: oh.. seems we lost the homepage field

File: plugins/krew.yaml [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
+  version: "v0.2.1-103-g0837beb"
```
Comment (corneliusweig) [inline plugins/krew.yaml]: :man_facepalming: let me fix this..

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **doesn't** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file plugins/krew.yaml is not a straightforward version bump: changing old version ("0.2.1") with new version ("0.3.0") in the url (https://storage.googleapis.com/krew/v0.2.1/krew.tar.gz) did not result in the new url (https://github.com/kubernetes-sigs/krew/releases/download/v0.3.0/krew.tar.gz), expected: https://storage.googleapis.com/krew/v0.3.0/krew.tar.gz

File: plugins/krew.yaml (hunk 1 above)
Comment (ahmetb) [reply plugins/krew.yaml]: let's not fix .yaml here, we can re-tag v0.3.0 in the main repo

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **doesn't** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file plugins/krew.yaml is not a straightforward version bump: changing old version ("0.2.1") with new version ("0.3.0") in the url (https://storage.googleapis.com/krew/v0.2.1/krew.tar.gz) did not result in the new url (https://github.com/kubernetes-sigs/krew/releases/download/v0.3.0/krew.tar.gz), expected: https://storage.googleapis.com/krew/v0.3.0/krew.tar.gz

**Comment by ahmetb** [comment]
/lgtm
/approve
:shipit:

**Comment by corneliusweig** [comment]
It's funny how we don't trust our own code:
```
Upgraded plugin: krew
WARNING: You installed a plugin from the krew-index plugin repository.
   These plugins are not audited for security by the Krew maintainers.
   Run them at your own risk.
```
`¯\_(^^)_/¯`

**Pull request merged**